builder/js/ncf.js

```diff
@@ -1149,7 +1159,30 @@ $scope.onImportFileChange = function (fileEl) {
           // We will lose the link between the selected method and the technique, to prevent unintended behavior, close the edit method panel
           $scope.selectedMethod = undefined;
         }
-        $scope.resetFlags();
+        $scope.resetFlags();i
```

Comment:
There is a 'i' after the semi-colon here!!

Reply:
:eye: :eye_speech_bubble: :eyes: :bird:

Reply:
JS is so forgiving

Reply:
i does nothing, you only call a variables that is undefined ... but yes ...